src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -112,7 +112,21 @@ Each Ticket should be generated from the prior one in the ticket-chain and verif
 {{< readfile file="storage_power_consensus_subsystem.id" code="true" lang="go" >}}
 {{< readfile file="storage_power_consensus_subsystem.go" code="true" lang="go" >}}
 
-### Repeated Leader Election attempts
+## Repeated Leader Election attempts
 
 In the case that no miner is eligible to produce a block in a given round of EC, the storage power consensus subsystem will be called by the block producer to attempt another leader election by incrementing the nonce appended to the ticket drawn from the past in order to attempt to find a new winning `PartialTicket` and trying again. 
-Note that a miner may attempt to grind through tickets by incrementing the nonce repeatedly until they find a winning ticket. However, any block so generated in the future will be rejected by other miners (with synchronized clocks) until that epoch's appropriate time.
\ No newline at end of file
+Note that a miner may attempt to grind through tickets by incrementing the nonce repeatedly until they find a winning ticket. However, any block so generated in the future will be rejected by other miners (with synchronized clocks) until that epoch's appropriate time.
+
+{{<label min_miner_size>}}
+## Minimum Miner Size
+
+{{% notice placeholder %}}
+The below values are currently placeholders.
+{{% /notice %}}
+
+In order to secure Storage Power Consensus, the system defines a minimum miner size required to participate in consensus.
+
+Specifically, miners must have either at least 100TB of active power (i.e. storage power currently used in storage deals) or 1/3 of the network's active storage power to participate in SPC.
+Miners smaller than this cannot mine blocks and earn block rewards. Put another way, this means that network liveness requires at least 3 active miners (running storage deals) or 300TB of active power.
+
+Accordingly, to bootstrap the network, the genesis block must include 3 valid storage deals made between the genesis miners(along with appropriate committed storage).
```

Comment:
Is the 1/3 edge case just for the genesis block?  If so it seems like we could cheat for early testnet and transition to genesis miners with 100s of TBs

Reply:
Here again I agree with @ZenGround0 that a sole miner is automatically a valid block producer. Of course we don't expect to start mainnet that way, but it's fine for testnets.

Reply:
The point is rather more to ensure liveness in case the network shrinks too much. Ie no need to reboot FIL if there remains a single active participant. We intend to have large genesis miners. Will revise